packages/blob/package.json

```diff
@@ -19,7 +19,10 @@
   "exports": {
     ".": {
       "types": "./dist/src/lib.d.ts",
-      "browser": "./src/lib.js",
+      "browser": {
+        "require": "./dist/lib.cjs",
```

Comment:
Should this be `./dist/src/lib.cjs`?  Unsure why our output directory is setup that way but nothing is built to the top level in `dist/` and only `dist/src` is packaged for npm per package.json `files`

Reply:
This looks to be the case maybe for file/form-data too

Reply:
The ESM export is also pointing to the root of `dist`

https://github.com/remix-run/web-std-io/blob/5a3703c5c17523b9e8d39ad05eb58c3acc1734c3/packages/blob/package.json#L16
https://github.com/remix-run/web-std-io/blob/5a3703c5c17523b9e8d39ad05eb58c3acc1734c3/packages/blob/package.json#L24

Reply:
I don't think those are pointing to the root of `dist`, they're pointing to the root of the package, because the `src` directory is included in the tarball via `files`.

If you run `npm pack --dry-run`, you can see that there is no such file `./dist/lib.cjs`:

```
~/me/shopify/repos/web-std-io/packages/blob (export-CJS-for-browser)> npm pack --dry-run
npm notice
npm notice 📦  @remix-run/web-blob@3.0.5
npm notice === Tarball Contents ===
npm notice 1.1kB  License.md
npm notice 3.5kB  Readme.md
npm notice 8.1kB  dist/src/blob.cjs
npm notice 14.0kB dist/src/blob.cjs.map
npm notice 303B   dist/src/blob.d.ts
npm notice 144B   dist/src/blob.d.ts.map
npm notice 1.2kB  dist/src/blob.node.cjs
npm notice 1.1kB  dist/src/blob.node.cjs.map
npm notice 182B   dist/src/blob.node.d.ts
npm notice 128B   dist/src/blob.node.d.ts.map
npm notice 701B   dist/src/lib.cjs
npm notice 341B   dist/src/lib.cjs.map
npm notice 242B   dist/src/lib.d.ts
npm notice 103B   dist/src/lib.d.ts.map
npm notice 1.1kB  dist/src/lib.node.cjs
npm notice 881B   dist/src/lib.node.cjs.map
npm notice 192B   dist/src/lib.node.d.ts
npm notice 152B   dist/src/lib.node.d.ts.map
npm notice 600B   dist/src/package.cjs
npm notice 118B   dist/src/package.cjs.map
npm notice 150B   dist/src/package.d.ts
npm notice 111B   dist/src/package.d.ts.map
npm notice 1.8kB  package.json
npm notice 7.5kB  src/blob.js
npm notice 496B   src/blob.node.js
npm notice 164B   src/lib.js
npm notice 617B   src/lib.node.js
npm notice 111B   src/package.js
npm notice === Tarball Details ===
npm notice name:          @remix-run/web-blob
npm notice version:       3.0.5
npm notice filename:      remix-run-web-blob-3.0.5.tgz
npm notice package size:  9.8 kB
npm notice unpacked size: 45.1 kB
npm notice shasum:        7b1929f2dc4a05ae0d41c9dcd0fb1203562ec782
npm notice integrity:     sha512-G+1cRj9Ho+2YS[...]PuxMsM9/rCQ/A==
npm notice total files:   28
npm notice
remix-run-web-blob-3.0.5.tgz
```

Reply:
Oh I must have probably copied it over from `fetch` 🙈

https://github.com/remix-run/web-std-io/blob/5a3703c5c17523b9e8d39ad05eb58c3acc1734c3/packages/fetch/package.json#L14